public/calendar.css

```diff
@@ -6,6 +6,63 @@ body{
   top: 0; 
 }
 
+.btn {
+  /*display: inline-block;*/
+  padding: 6px 12px;
+  margin-bottom: 0;
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 1.42857143;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: middle;
+  -ms-touch-action: manipulation;
+      touch-action: manipulation;
+  cursor: pointer;
+  -webkit-user-select: none;
+     -moz-user-select: none;
+      -ms-user-select: none;
+          user-select: none;
+  background-image: none;
+  border: 1px solid transparent;
+  border-radius: 4px;
+}
+
+#user-container {
+  display: flex;
```

Comment:
My understanding is that this isn't cross platform yet and that you have to use the -moz and -webkit versions in addition to this. Maybe look into that?